setup.py

```diff
@@ -24,7 +24,6 @@
 version = '6.8.dev0'
 
 INSTALL_REQUIRES = [
-    'setuptools',
```

Comment:
Maybe we want to still depend on it? 🤔

Reply:
I guess we do still need it for `pkg_resources` - sigh.  I've put it back.